release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -105,6 +105,17 @@ For more information, refer to xref:../networking/ovn_kubernetes_network_provide
 
 The containerized Data Plane Development Kit (DPDK) is now GA in {product-title} {product-version}. For more information, see xref:../networking/hardware_networks/using-dpdk-and-rdma.adoc#using-dpdk-and-rdma[Using virtual functions (VFs) with DPDK and RDMA modes].
 
+[id="ocp-4-9-networking-metallb"]
+==== MetalLB load balancer
+
+This release introduces the MetalLB Operator. After installing and configuring the MetalLB Operator, you can deploy MetalLB to provide a native load balancer implementation for services on bare-metal clusters. Other on-premise infrastructures that are like bare metal might also benefit.
```

Comment:
Is there a reason for hyphenating "bare-metal" on the first reference? 

Suggested edit: 
s/might/can

Reply:
IIUC, bare metal is used as an adjective to modify the noun, clusters.  I'm attempting to follow the guidance from <https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal-adj>.